doc/user/content/sql/begin.md

```diff
@@ -27,7 +26,7 @@ Transactions in Materialize do not support interleaving arbitrary kinds of state
 
 ### Read-only transactions
 
-A **read-only** transaction starts with a [`SELECT`](/sql/select) statement and allows only `SELECT` statements.
```

Comment:
The `READ ONLY` syntax and a "Read-only transaction" are slightly different concepts and I think the current wording is more accurate than this diff. All txns in mz are either read-only, write-only, or "a single SUBSCRIBE". We decide which txn type based on the first statement a user attempts to execute. The `READ ONLY` syntax prevents a write-only txn from starting.

I think this diff chunk should be reverted.

Reply:
Done